spec/classes/opendkim_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'spec_helper'
+
+describe 'opendkim' do
+  on_supported_os.each do |os, _facts|
+    context "on #{os}" do
+      let(:facts) { { 'os' => { 'name' => 'RedHat' } } }
```

Comment:
```suggestion
    let(:facts) do
      facts
    end
```

Reply:
Thanks @Ubeek ,for this suggestion it did worked in the facts collection with the Rhel and Centos